src/collar/oc_anim.lsl

```diff
@@ -222,15 +223,15 @@ float g_fStandHover=0;
 list g_lAdjustments;
 float g_fAdjustment = 0.02;
 integer g_iStoppedAdjust;
-
+string g_pose = "";

```

Comment:
I'm not seeing fully where this variable is even used.

Reply:
it basically prevents multiple pose animations from being pushed when poses are changed, since they don't get explicitly stopped first when you just select a new one from the menu.